src/js/outreach/views/dialog_views.js

```diff
@@ -15,7 +15,7 @@ const DialogView = View.extend({
   `,
   templateContext() {
     return {
-      name: auth0Config.name,
+      name: appConfig.name,
```

Comment:
Use this instead of `auth0Config.name`? I don't know the reason for using that originally.

And I couldn't get `auth0Config.name` to work in cypress.

Reply:
oh yeah this changed recently